### PR TITLE
[AutoML] Bump version to ML.NET 1.3.1 in AutoML API and CLI and AutoML package version to 0.15.1

### PR DESCRIPTION
**Pull request by srsaggam**
Bumping ml.net version to 1.3.1
Needs to be smoke tested by : @justinormont  Thanks.

### CODE REVIEW

**Comment by srsaggam** [comment]
@justinormont @daholste  @Dmitry-A

File: src/mlnet/Templates/Console/ModelProject.cs [hunk 1]
```diff
@@ -27,15 +27,15 @@ public virtual string TransformText()
         {
             this.Write("<Project Sdk=\"Microsoft.NET.Sdk\">\r\n\r\n  <PropertyGroup>\r\n    <TargetFramework>netc" +
                     "oreapp2.1</TargetFramework>\r\n  </PropertyGroup>\r\n  <ItemGroup>\r\n    <PackageRefe" +
-                    "rence Include=\"Microsoft.ML\" Version=\"1.2.0\" />\r\n");
+                    "rence Include=\"Microsoft.ML\" Version=\"1.3.1\" />\r\n");
```
Comment (Dmitry-A) [inline src/mlnet/Templates/Console/ModelProject.cs]: ugh.  can you make it a const somewhere so it's not duplicated all over the place?

File: build/BranchInfo.props [hunk 2]
```diff
@@ -28,7 +28,7 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(IsStableProject)' != 'true'">
     <MajorVersion>0</MajorVersion>
-    <MinorVersion>15</MinorVersion>
+    <MinorVersion>16</MinorVersion>
     <PatchVersion>0</PatchVersion>
```
Comment (justinormont) [inline build/BranchInfo.props]: ML.NET was released at `1.3.1` & `0.15.1` for preview. So this preview version should match at `0.15.1`.

After we release, we'll bump our version to 0.16.0. Same as ML.NET did after releasing ([PR](https://github.com/dotnet/machinelearning/pull/4069)).

Ordering:
* Set AutoML/CLI version to 0.15.1
* Update the AutoML/CLI to use ML.NET `1.3.1`
* _(release)_
* Update AutoML/CLI version to 0.16.0

Comment (justinormont) [reply build/BranchInfo.props]: Thanks.

#resolved

**Pull request merged**